tests/test_tools.py

```diff
@@ -55,6 +57,32 @@ async def invalid_tool(ctx: RunContext[None]) -> str:  # pragma: no cover
     )
 
 
+def test_builtin_tool_registry():
+    """
+    Test that built-in functions can be registered as tools.
+    """
+
+    with pytest.raises(UserError, match='Error generating schema for'):
```

Comment:
Please include the actual new error message we added `Error getting signature for`. `match` take a regex